### DISCUSSION TITLE
Copilot workspace accessing

### DISCUSSION THREAD

**Discussion by gauravpanchal09** [community/community#101786] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Steps or point of to access GitHub copilot workspace?
Is a app, regis. process to request access to copilot workspace?

Thanks

**Comment by mayankzarkar** [comment, marked as answer]
Hello @gauravpanchal09 

You can find your answer in below link.
https://github.com/orgs/community/discussions/87460#discussioncomment-8150416

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @gauravpanchal09 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you.
